mm2src/coins/utxo/bchd_grpc.rs

```diff
@@ -79,11 +80,29 @@ pub struct ValidateSlpUtxosErr {
     kind: ValidateSlpUtxosErrKind,
 }
 
+impl From<ValidateSlpUtxosErr> for ValidatePaymentError {
+    fn from(err: ValidateSlpUtxosErr) -> Self {
+        match err.kind {
+            ValidateSlpUtxosErrKind::MultiReqErr(_) => Self::Transport(err.to_string()),
+            ValidateSlpUtxosErrKind::InvalidSlpTxData(_) => Self::TxDeserializationError(err.to_string()),
```

Comment:
Transaction is deserialized successfully in this case, so `WrongPaymentTx` seems more suitable here.

Reply:
done